docs/jwst/pipeline/calwebb_spec3.rst

```diff
@@ -80,8 +95,8 @@ Source-based calibrated data
 
 For NIRSpec fixed-slit, NIRSpec MOS, and NIRCam and NIRISS WFSS, which have a defined
 set of slits or sources, the data from the input calibrated exposures is reorganized
-by :ref:`exp_to_source <exp_to_source>` so that all of the instances of data for a
-particular source/slit are contained in a
+by the :ref:`exp_to_source <exp_to_source>` step so that all of the instances of data
```

Comment:
Just a question:  Why is "_step" included within the angle brackets for other step names but not for this one?

Reply:
Just randomness of how the cross-reference labels were created/spelled in the corresponding rst files for the steps.